src/rod/sdf.py

```diff
@@ -0,0 +1,68 @@
+import dataclasses
+import pathlib
+from typing import List, Optional, Union
+
+import mashumaro
+import xmltodict
+
+from .element import Element
+from .model import Model
+from .world import World
+
+
+@dataclasses.dataclass
+class Sdf(Element):
+
+    version: str = dataclasses.field(metadata=mashumaro.field_options(alias="@version"))
```

Comment:
Could it make sense to check this value at some point to raise an error/warning if the version is not 1.7 ?

Reply:
Makes sense, done in https://github.com/ami-iit/rod/pull/1/commits/a785e8e422f4b92288946286adcba76e815022bd